tool/src/test/java/org/wildfly/security/tool/VaultCommandTest.java

```diff
@@ -69,12 +121,24 @@ public void singleConversionBasicTest() throws Exception {
      */
     @Test
     public void bulkConversionBasicTest() throws Exception {
-        String[] args = {"--bulk-convert", "target/test-classes/bulk-vault-conversion-desc"};
+        String[] args;
+        if(IS_IBM) {
+            args = new String[]{"--bulk-convert", "target/test-classes/bulk-vault-conversion-desc-ibm"};
+        }
+        else {
```

Comment:
Just a very minor code style comment, the ```else {``` line here should be moved to the previous line:

```
if (..) {
 ...
} else {
 ...
}
```

Reply:
Fixed